### ISSUE TITLE
ledger-default-date-format setting not respected in reports

### ISSUE THREAD

**Issue by doolio** [ledger/ledger-mode#289]
I've set this to match `ledger-iso-date-format`. The docstring `ledger-default-date-format` suggests the set format will be used by ledger throughout.  I took this to also include reports. It does not. At a minimum I would like the report dates to include the full year e.g. 2021 vs 21.

Perhaps if/when [ledger/issues/1853](https://github.com/ledger/ledger/issues/1853) is implemented this would be respected?

Thanks for your time.

**Comment by bcc32** [comment]
Yeah, it looks like reports are the one place where ledger-mode might call ledger but not go through the `ledger-exec-ledger` function (I guess it's because reports could be arbitrary shell commands, not just a single invocation of ledger with some args).

(I don't think the ledger-cli issue is related.)
